### ISSUE TITLE
Can not create dashboard

### ISSUE THREAD

**Issue by lkoniecki** [dynatrace-oss/dynatrace-gcp-function#8]
I'm getting the following error when script creates dashboards in my tenant:
```
- Create ["Google Cloud APIs"] dashboard from file [dashboards/api.json]
{"error":{"code":400,"message":"Could not map JSON at 'tiles[2].queries[0].filterBy' near line 53 column 25"}}
- Create ["Google Cloud Function"] dashboard from file [dashboards/cloud_function.json]
{"error":{"code":400,"message":"Could not map JSON at 'tiles[1].queries[0].filterBy' near line 43 column 25"}}
- Create ["Google Cloud SQL"] dashboard from file [dashboards/cloudsql_database.json]
{"error":{"code":400,"message":"Could not map JSON at 'tiles[0].queries[0].filterBy' near line 31 column 25"}}
- Create ["Google Cloud Datastore"] dashboard from file [dashboards/datastore.json]
{"error":{"code":400,"message":"Could not map JSON at 'tiles[0].queries[0].filterBy' near line 31 column 25"}}
- Create ["Google Cloud Filestore"] dashboard from file [dashboards/filestore_instance.json]
{"error":{"code":400,"message":"Could not map JSON at 'tiles[3].queries[0].filterBy' near line 73 column 25"}}
- Create ["Google Cloud Storage"] dashboard from file [dashboards/gcs_bucket.json]
{"error":{"code":400,"message":"Could not map JSON at 'tiles[1].queries[0].filterBy' near line 43 column 25"}}
- Create ["Google Cloud Load Balancing"] dashboard from file [dashboards/lb.json]
{"id":"5d7c0de3-1344-4216-a755-313a3bc30892","name":"Google Cloud Load Balancing"}
- Create ["Google Cloud Pub/Sub"] dashboard from file [dashboards/pubsub.json]
{"error":{"code":400,"message":"Could not map JSON at 'tiles[1].queries[0].filterBy' near line 43 column 25"}}
```

**Comment by pawelsiwek** [comment]
We are working on it and keep You in loop.

**Comment by pawelsiwek** [comment]
The issue is fixed with release-0.0.12
